src/scipp/html/formatting_html.py

```diff
@@ -17,17 +17,14 @@
 SPARSE_PREFIX = "len={}"
 
 
-def _format_array(data, size, ellipsis_after, do_ellide=True):
+def _format_array(data, size, ellipsis_after, do_elide=True):
     i = 0
     s = []
     while i < size:
-        if do_ellide and i == ellipsis_after and size > 2 * ellipsis_after + 1:
+        if do_elide and i == ellipsis_after and size > 2 * ellipsis_after + 1:
             s.append("...")
             i = size - ellipsis_after
         elem = data[i]
-        if hasattr(elem, "__round__"):
-            if not hasattr(data, "dtype") or data.dtype != bool:
-                elem = round(elem, 2)
```

Comment:
Where do we control the number of decimal places now?

Reply:
We don't. Just like in `__str__`, it uses whatever the underlying implementation thinks is good.

Reply:
I don't think that is going to work: It will take too much horizontal space it will not be readable in many cases. This is exactly why this specific code was introduced, so we'll need something similar that limits the total space taken as far as possible.